src/components/LayoutWithRouting.tsx

```diff
@@ -38,6 +55,7 @@ const LayoutWithRouting: React.FC = () => {
     <>
       <GlobalStyle additionalStyles={globalStyles} />
       <Navbar default />
```

Comment:
Just a thought now. Maybe we could rename the `Navbar` to `Topbar` or something? Now the navigation is in the `Sidenav`, so this way we can avoid some confusion